cmd/traffic_cache_tool/CacheTool.cc

```diff
@@ -456,6 +559,16 @@ Stripe::dir_init_segment(int s)
   }
 }
 
+Errata
+Stripe::loadDir()
+{
+  Errata zret;
+  char *raw_dir = (char *)ats_memalign(ats_pagesize(), this->vol_dirlen());
+  dir           = (CacheDirEntry *)(raw_dir + this->vol_headerlen());
+  // read directory
+  pread(this->_span->_fd, raw_dir, this->vol_dirlen(), this->_start);
```

Comment:
Check the return of `pread`? Especially for the CacheTool, it must check *everything*.